entc/gen/template/base.tmpl

```diff
@@ -307,7 +307,7 @@ func withHooks[V Value, M any, PM interface {
 		return exec(ctx)
 	}
 	var mut Mutator = MutateFunc(func(ctx context.Context, m Mutation) (Value, error) {
-		mutationT, ok := m.(PM)
+		mutationT, ok := any(m).(PM) // redundant any() conversion is a workaround for golang/go#58633
```

Comment:
Change it to a template comment, please?
```suggestion
		{{- /* redundant any() conversion is a workaround for golang/go#58633 */}}
		mutationT, ok := any(m).(PM)
```

Reply:
Great! Completely forgot about them :)
Also did not like that the result gets into the generated code.
Fixed